ethereum_history_api/oracles/src/script/noir_fixtures/storage_proof.ts

```diff
@@ -0,0 +1,50 @@
+import { Hash } from 'viem';
+import { encodeHexString } from '../../noir/noir_js/encode.js';
+import { encodeProof } from '../../noir/oracles/accountOracles.js';
+import { encodeBytes32 } from '../../noir/oracles/encode.js';
+
+export function createStorageProofFixture(
+  storageRoot: Hash,
+  storageProofs: { key: Hash; proof: Hash[]; value: bigint }[]
+): string {
+  const storageProofsNoir = storageProofs.map(createSingleStorageProofFixture);
+  const encodedStorageRoot = encodeHexString(storageRoot);
+  return `
+use crate::storage::{StorageProof, StorageProofsWithStorageRoot};
+use crate::misc::types::Bytes32;
+use crate::account::PROOF_LEN;
+
+global storage_root: Bytes32 = [${encodedStorageRoot}];
+
+global proofs: [StorageProof; ${storageProofs.length}] = [
+  ${storageProofsNoir.join(',\n')}
+];
+
+global storage_proofs_with_storage_root = StorageProofsWithStorageRoot {
+  storage_root,
+  proofs
+};
+`;
+}
+
+function createSingleStorageProofFixture(storageProof: { key: Hash; proof: Hash[]; value: bigint }): string {
+  const key = encodeHexString(storageProof.key);
+  const value = encodeBytes32(storageProof.value);
+  const proof = encodeProof(storageProof.proof).map((byte) => parseInt(byte, 16));
+  const depth = storageProof.proof.length;
+  const storageProofFixture = `
+  StorageProof {
```

Comment:
Is state proofs we have those as separate variables and then create a common object. 
```rust
use crate::account::StateProof;

global key = [...];
global value = [...];
global proof = [...];
global depth = 9;

global state_proof = StateProof {key, value, proof, depth};

```
Here it's inconsistent.
I've checked the tests and we never use those separate values, so I agree - that it's better to have it inline for readability, but let's change state fixtures too then

Reply:
Done also in state fixtures. Will update account and header in separate PR